### PR TITLE
Ae 857 datadog node lib

### PR DESCRIPTION
**Pull request by manikant16**

### CODE REVIEW

File: packages/datadog-metrics-tracker/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Influx Metrics Tracker
+
+Track metrics and store them in an DataDog.
```
Comment (MelCBuckOVO) [inline packages/datadog-metrics-tracker/README.md:3]: --> in DataDog

File: packages/datadog-metrics-tracker/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+## Using
+TODO
```
Comment (MelCBuckOVO) [inline packages/datadog-metrics-tracker/README.md:6]: Please do it : )

File: packages/datadog-metrics-tracker/package.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@ovotech/datadog-metrics-tracker",
+  "description": "Track metrics and store them in an DataDog",
```
Comment (MelCBuckOVO) [inline packages/datadog-metrics-tracker/package.json:3]: in DataDog

File: packages/datadog-metrics-tracker/package.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  "version": "1.3.5",
+  "main": "dist/index.js",
+  "source": "src/index.ts",
+  "types": "dist/index.d.ts",
+  "author": "Hayden Field <hayden.field@ovoenergy.com>",
```
Comment (MelCBuckOVO) [inline packages/datadog-metrics-tracker/package.json:8]: you are the author Mani : )

**Review by MelCBuckOVO — COMMENTED** [review]
looks good to me - just a few cosmetic tweaks please

we don't need to commit yarn-error log - please delete it

File: packages/datadog-metrics-tracker/README.md (hunk 1 above)
Comment (manikant16) [reply packages/datadog-metrics-tracker/README.md:3]: Updated

File: packages/datadog-metrics-tracker/README.md (hunk 2 above)
Comment (manikant16) [reply packages/datadog-metrics-tracker/README.md:6]: Updated

File: packages/datadog-metrics-tracker/package.json (hunk 4 above)
Comment (manikant16) [reply packages/datadog-metrics-tracker/package.json]: Updated Thanks :-)

**Review by MelCBuckOVO — APPROVED** [review]

**Review by dmcregula — APPROVED** [review]

**Pull request merged**